lib/parallel_cucumber/job.rb

```diff
@@ -0,0 +1,13 @@
+module ParallelCucumber
+  class Job
+    RUN_TESTS = 'run_tests'.freeze
+    PRECHECK = 'precheck'.freeze
+    DIE = 'die'.freeze
```

Comment:
Maybe use symbols? I don't mind to have strings anyway..

Reply:
changed to symbols